one-ui/src/main/java/com/marklogic/hub/oneui/managers/ModelManager.java

```diff
@@ -154,4 +157,43 @@ public JsonNode getModel(String modelName) {
 
         return handle.get();
     }
+
+    /**
+     * Get latest job info for specified model
+     *
+     * @param modelName Name of the entity model
+     * @return - a JsonNode containing job info
+     */
+    public JsonNode getLatestJobInfo(String modelName) {
+        return getJobInfoFromDB(finalDataServiceClient, modelName);
+    }
+
+    /**
+     * Get latest job info for all models
+     *
+     * @return - a list of JsonNode containing job info
+     */
+    public List<JsonNode> getLatestJobInfoForAllModels() {
+        return getModelNames()
+            .stream()
+            .map(s -> getJobInfoFromDB(finalDataServiceClient, s))
+            .filter(Objects::nonNull)
+            .collect(Collectors.toList());
+    }
+
+    JsonNode getJobInfoFromDB(DatabaseClient dbClient, String modelName) {
+        JsonNode node = null;
+        try {
+            node = JobInfo.on(dbClient).getLatestJobData(modelName);
+        }
+        catch (MarkLogicServerException e) {
+            /*
```

Comment:
Why would the user not be authorized? I'm wondering more generally - why would we call this method if the user isn't authorized? This seems like something we'd want to handle via authorities - I think Spring Security has a method for checking this programmatically so an exception isn't thrown right away - e.g. https://stackoverflow.com/questions/3021200/how-to-check-hasrole-in-java-code-with-spring-security . 

I think this needs to be changed because if this exception occurs for some other reason, we don't necessarily want to just log it and move on.

Reply:
Explore has two users, the architect user that can only access the entity models but not the entity instances and the user that can access both the entity models and instances.
The explorer architect user won't have access to this job info data service endpoint and in that case, the UI does not show any data in the last harmonized timestamp column. 

Now, that we have authorities we can rely on them to hide that column from the UI. 

I'll just create a task to check for the right authority for now as this would require additional work on the UI to hide the extra column for an explorer architect user. Making the change just on the backend will break the app as the frontend would not handle the 403 access denied when this call is made.

We can take up this task up once @brucean52 completes the task of merging the frontend explorer code with OneUI.